src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__image"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+      <p class="product-card__description">
+        Моноблок APPLE A1419 iMac<br>
+        27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="product-card__code">
+        Код товара: 195434
+      </p>
+      <div class="product-card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="product-card__reviews">Отзывов: 5 </p>
+      </div>
+      <div class="product-card__price">
+        <p class="product-card__price-title">Цена:</p>
+        <p class="product-card__price-number">69 999 грн</p>
+      </div>
+      <a
+        class="product-card__button"
+        data-qa="hover"
+        href="/"
+      >
+        Купить
+      </a>
+    </div>
+  </body>
 </html>
```

Comment:
your style.scc file is empty

Reply:
you are right, made changes, now it is there)